plugin/src/main/java/io/jenkins/plugins/analysis/core/util/LogHandler.java

```diff
@@ -72,8 +80,10 @@ private PluginLogger createLogger(final TaskListener listener, final String name
      *         the issues with the collected logging messages
      */
     public void log(final Report report) {
-        logErrorMessages(report);
-        logInfoMessages(report);
+        if (!quiet) {
+            logErrorMessages(report);
+            logInfoMessages(report);
+        }
```

Comment:
This guard should be better used for printing a individual line, otherwise logging using `log` (line 100) does not work.